installer/Languages/README.md

```diff
@@ -0,0 +1,5 @@
+# Language Files
+
+### All language files used in inno are archived here.
+
+If the file name has `(old)` at the end, this language file can be used for `inno setup 5`.
```

Comment:
you are right.If use `inno6`, we really don't need these old files.
But if delete, it may make the whole `Sakura` project unable to compile in `inno5`.It may be necessary to modify the min compilation environment.

Reply:
I see, fair point. Then there has to be a decision among the project contributors. Please be patient with us for that. :)